isso/tests/test_comments.py

```diff
@@ -844,6 +864,64 @@ def testModerateComment(self):
         # Comment should no longer exist
         self.assertEqual(self.app.db.comments.get(id_), None)
 
+    def testPendingWithoutAdmin(self):
+        self.conf.set("admin", "enabled", "false")
```

Comment:
Are these tests guaranteed to be run in order? I'd prefer you un-set the `admin.enabled` value at test end.